### PR TITLE
add kd en doc

### PR DESCRIPTION
**Pull request by littletomatodonkey**
add doc of knowledge distillation

### CODE REVIEW

File: docs/en/algorithm_introduction/knowledge_distillation.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+
+<a name='2'></a>
+
+## 2. Application of knowledge distillation
+
+Knowledge distillation algorithm is widely used in lightweight tasks. For tasks that need to meet specific accuracy, by using the knowledge distillation method, we can achieve the required accuracy with a smaller model, thereby reducing model deployment cost.
+
+
+What's more, for the same model structure, pre-trained models obtained by knowledge distillation often performs better, and these pre-trained models can also improve performance of the downstream tasks. For example, in , a pre-trained image classification model with higher accuracy can also help other tasks obtain significant accuracy gains such as target detection, image segmentation, OCR, and video classification.
```
Comment (TingquanGao) [inline docs/en/algorithm_introduction/knowledge_distillation.md]: For example, ~~in ,~~ a pre-trained image classification model ...

File: docs/en/algorithm_introduction/knowledge_distillation.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+<a name='3'></a>
+
+## 3. Overview of knowledge distillation methods
+
+<a name='3.1'></a>
+
+### 3.1 Response based distillation
+
+
+Knowledge distillation algorithm is firstly proposed by Hinton, which is called KD. In addition to base cross entropy loss, KL divergence loss between output of student model and teacher model is also added onto the total training loss. It's noted that a larger teacher model is needed to guide the training process of the student model.
```
Comment (TingquanGao) [inline docs/en/algorithm_introduction/knowledge_distillation.md]: onto -> into

File: docs/en/algorithm_introduction/knowledge_distillation.md (hunk 1 above)
Comment (littletomatodonkey) [reply docs/en/algorithm_introduction/knowledge_distillation.md]: done, thx

File: docs/en/algorithm_introduction/knowledge_distillation.md (hunk 2 above)
Comment (littletomatodonkey) [reply docs/en/algorithm_introduction/knowledge_distillation.md]: done, thx

**Review by TingquanGao — APPROVED** [review]

**Pull request merged**